x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Popover/ServiceMetricList.tsx

```diff
@@ -106,23 +112,27 @@ export function ServiceMetricList({
         : null
     }
   ];
+  const showBadgeRow = frameworkName || (numInstances && numInstances > 1);
```

Comment:
Is it necessary to check whether `numInstances` is defined? 
```suggestion
  const showBadgeRow = frameworkName || numInstances > 1;
```

Reply:
Updated in 946f1854b6. Also removed the undefined check on line 125. I think it was optional at one point. Thanks for catching!